### DISCUSSION TITLE
Organizing course list in timetable page

### DISCUSSION THREAD

**Discussion by TiaMcDowell** [VikeLabs/courseup#229] [Ideas]
When viewing my timetable the courses are organized  by when they were added. It would be nice if I could organize them by other filters like course level or department. It would be even better if I could click and drag to rearrange the courses I am looking at, and possibly even change the colour.

**Comment by keithradford** [comment]
I've used https://github.com/atlassian/react-beautiful-dnd before which I think could probably be used here. A few things to consider before implementing it but worth putting on the radar, thanks for the suggestion!
